L1 Welcome to EV101 course/About Discord.md

```diff
@@ -0,0 +1,67 @@
+# Thriving in Our Community
```

Comment:
@vsharma2288 This `#` should be `##` and same for remaining. You said in the comment on this PR that you have fixed this?